clientgui/DlgAdvPreferences.cpp

```diff
@@ -670,7 +670,10 @@ bool CDlgAdvPreferences::ValidateInput() {
     wxString invMsgLimit10 = _("Number must be between 0 and 10");
     wxString invMsgLimit100 = _("Number must be between 0 and 100");
     wxString invMsgLimit1_100 = _("Number must be between 1 and 100");
+    wxString invMsgIdle = _("Warning:  'When to suspend' time settings will not allow any CPU and/or GPU tasks to compute.  Reduce time to compute after idle or increase time to suspend after idle.");
     wxString buffer;
+    wxString bufferPIF;  //variable to store Processor Idle For string.
+    wxString bufferNRI;  //variable to store No Recent Input string.
     double startTime, endTime;
```

Comment:
I understand.  Because the "suspend when no mouse/keyboard input in last" variable is always enabled, I should just use the usual "buffer" variable for "bufferNRI".  BufferPIF can be initialized in the conditional statement.

Your suggested change is not related to the code I modified, but I see your point, I can initialize these variables.